packages/webcert/src/feature/list/filter/DateRangeFilter.tsx

```diff
@@ -166,18 +166,18 @@ const DateRangeFilter: React.FC<Props> = ({ config, onChange }) => {
     return savedValue ? savedValue.to : ''
   }
 
-  const onValidationError = useCallback(
-    (isInactive: boolean, validationError: ValidationError) => {
-      if (validationError.field === configTo.id) {
-        setToValidationError(isInactive ? null : validationError)
-        dispatch(updateValidationError({ id: configTo.id, value: !isInactive }))
-      } else {
-        setFromValidationError(isInactive ? null : validationError)
-        dispatch(updateValidationError({ id: configFrom.id, value: !isInactive }))
-      }
-    },
-    [configFrom.id, configTo.id, dispatch]
-  )
+  // const onValidationError = useCallback(
+  //   (isInactive: boolean, validationError: ValidationError) => {
+  //     if (validationError.field === configTo.id) {
+  //       setToValidationError(isInactive ? null : validationError)
+  //       dispatch(updateValidationError({ id: configTo.id, value: !isInactive }))
+  //     } else {
+  //       setFromValidationError(isInactive ? null : validationError)
+  //       dispatch(updateValidationError({ id: configFrom.id, value: !isInactive }))
+  //     }
+  //   },
+  //   [configFrom.id, configTo.id, dispatch]
+  // )
```

Comment:
Maybe remove this commented out section?

Reply:
Good catch, I had forgotten about the list validations, so this needed some more work. Should be fixed now.